stingray/crossspectrum.py

```diff
@@ -121,6 +130,17 @@ def _make_crossspectrum(self, lc1, lc2):
         if not isinstance(lc2, lightcurve.Lightcurve):
             raise TypeError("lc2 must be a lightcurve.Lightcurve object")
 
+        # Then check that GTIs make sense
+        if self.gti is None:
+            self.gti = cross_two_gtis(lc1.gti, lc2.gti)
+
+        if self.gti.shape[0] != 1:
```

Comment:
No, I'm all for being strict, too! I'm wondering if it's worth adding a reminder to the error message, though. Something along the lines of "If your GTI is contiguous, please check whether it is supplied in a 2D array (see documentation for details!". 

I have the feeling this is a problem that a lot of people would stumble over at least once.